assets/src/edit-story/components/canvas/page.js

```diff
@@ -23,158 +23,83 @@ const Background = styled.div.attrs( { className: 'container' } )`
-	z-index: 2;
-	left: ${ ( { x } ) => `${ x }px` };
-	top: ${ ( { y } ) => `${ y }px` };
-	width: ${ ( { width } ) => `${ width }px` };
-	height: ${ ( { height } ) => `${ height }px` };
-	transform: ${ ( { rotationAngle } ) => `rotate(${ rotationAngle }deg)` };
-	position: absolute;
-`;
-
 const Element = styled.div`
 	cursor: pointer;
 	user-select: none;
 `;
 
 function Page() {
-	const [ targetEl, setTargetEl ] = useState( null );
-
-	const {
-		state: { currentPage, hasSelection, selectedElements },
-		actions: { clearSelection, selectElementById, setPropertiesOnSelectedElements, toggleElementIdInSelection },
-	} = useStory();
 	const {
 		actions: { setBackgroundClickHandler },
 	} = useCanvas();
-	const handleSelectElement = useCallback( ( id ) => ( evt ) => {
+
+	const {
+		state: { currentPage, selectedElements },
+		actions: { clearSelection, selectElementById, toggleElementIdInSelection },
+	} = useStory();
+
+	const [ targetEl, setTargetEl ] = useState( null );
+	const [ pushEvent, setPushEvent ] = useState( null );
+
+	useEffect( () => {
+		setBackgroundClickHandler( () => clearSelection() );
+	}, [ setBackgroundClickHandler, clearSelection ] );
+
+	const handleSelectElement = useCallback( ( elId, evt ) => {
 		if ( evt.metaKey ) {
-			toggleElementIdInSelection( id );
+			toggleElementIdInSelection( elId );
 		} else {
-			selectElementById( id );
+			selectElementById( elId );
 		}
 		evt.stopPropagation();
-	}, [ toggleElementIdInSelection, selectElementById ] );
-	const selectionProps = hasSelection ? getUnionSelection( selectedElements, 0 ) : {};
-	useEffect( () => {
-		setBackgroundClickHandler( () => clearSelection() );
-	}, [ setBackgroundClickHandler, clearSelection ] );
 
-	let displayMoveable = false;
-	const frame = {
-		translate: [ 0, 0 ],
-		rotate: selectionProps.rotationAngle,
-	};
+		// @todo That's not necessary for multi-selection.
+		if ( 'pointerdown' === evt.type ) {
+			evt.persist();
+			setPushEvent( evt );
+		}
+	}, [ toggleElementIdInSelection, selectElementById ] );
 
-	const setStyle = ( target ) => {
-		target.style.transform = `translate(${ frame.translate[ 0 ] }px, ${ frame.translate[ 1 ] }px) rotate(${ frame.rotate }deg)`;
-	};
+	const singleSelection = 1 === selectedElements.length;
 
 	return (
 		<Background>
 			{ currentPage && currentPage.elements.map( ( { type, id, ...rest } ) => {
 				const comp = getComponentForType( type );
 				const Comp = comp; // why u do dis, eslint?
+
 				// Ignore multi-selection for now.
-				if ( 1 === selectedElements.length && selectedElements[ 0 ].id === id ) {
-					displayMoveable = true;
-				}
+				const isSelected = selectedElements.length ? selectedElements[ 0 ].id === id : false;
 				return (
-					<Element key={ id } onClick={ handleSelectElement( id ) }>
-						<Comp { ...rest } />
+					<Element
+						key={ id }
+						onClick={ ( evt ) => handleSelectElement( id, evt ) }
+					>
+						<Comp
+							{ ...rest }
+							onPointerDown={ ( evt ) => {
```

Comment:
Actually, nothing prevents us from moving up `onPointerDown` to the `<Element>` above, right?

Reply:
Just that then the `targetEl` and the `target` of the `pushEvent` would be different, probably better to keep it in sync. If we'd do that, it would already make sense to add the wrapper element and change the `targetEl` target as well.